tools/sof_perf_analyzer.py

```diff
@@ -204,8 +205,29 @@ def process_kmsg_file():
                 next_line = next(f)
                 widget_id = next_line.split('|')[0].split(':')[-1].strip()
                 # convert to the same ID format in mtrace
-                widget_id = int('0x' + widget_id[-6:], 16)
+                try:
+                    widget_id = int('0x' + widget_id[-6:], 16)
+                except ValueError:
+                    print("no valid widget ID found, try with bind information")
+                    not_found[widget_name] = ppln_id
+                    continue
                 comp_name[str(Component(ppln_id, widget_id))] = widget_name
+            elif match_obj := re.search(r' bind ', line):
```

Comment:
it is not search 'bind', it is ' bind ', with current dmesg logs, all the search is valid
this is simple and works, how about keep as for now? since this works, it can keep implementation
simple.
![image](https://github.com/thesofproject/sof-test/assets/17277136/283ae7bf-bbd3-4be7-a1bc-ada2ffc42abb)

Reply:
> with current dmesg logs, all the search is valid

This is called "programming by coincidence"